src/buildkit/git.ts

```diff
@@ -110,4 +113,28 @@ export class Git {
     res.shortName = repoSplitBySlash[repoSplitBySlash.length - 1].replace('.git', '');
     return res;
   }
+
+  public static async commitDate(ref: string, gitDir: string | undefined = undefined): Promise<Date> {
```

Comment:
Should be implemented in https://github.com/docker/actions-toolkit/blob/main/src/git.ts, not the buildkit one.

Reply:
done